src/main/frontend/src/components/aggregates/DeleteWithDialog.svelte

```diff
@@ -26,7 +26,7 @@
 
 <DeleteButton title="Delete {type}" on:click={open} bind:color/>
 
-<Portal target=".s-app"> <!-- solves Dialog -->
+<Portal target=".s-app"> <!-- solves Dialog capturing problem -->
```

Comment:
Portal adds this Dialog to the root of the app (in the html)

Reply:
looks good